counterpoll/main.py

```diff
@@ -241,6 +242,45 @@ def disable():
     configdb.mod_entry("FLEX_COUNTER_TABLE", "PG_WATERMARK", fc_info)
     configdb.mod_entry("FLEX_COUNTER_TABLE", BUFFER_POOL_WATERMARK, fc_info)
 
+# ACL counter commands
+@cli.group()
+@click.pass_context
+def acl(ctx):
+    """  ACL counter commands """
+    ctx.obj = ConfigDBConnector()
```

Comment:
@stepanblyschak i am wondering how it works...usually we have to init obj with `ensure_object`

Reply:
Why to use ensure_object here?

Reply:
@stepanblyschak without this statement, you can't access ctx.obj because it doesn't exist...maybe it has changed in recent click versions. Just double check it.

Reply:
It works, ensure_object has other reasons to be used.

Reply:
@stepanblyschak ok